.circleci/config.yml

```diff
@@ -252,7 +252,7 @@ jobs:
           command: 'yarn workspace @requestnetwork/smart-contracts run deploy'
       - run:
           name: 'Test request-node'
-          command: 'yarn workspace @requestnetwork/request-node run test'
+          command: 'yarn workspace @requestnetwork/request-node run test --ci'
```

Comment:
why it's not added here and in some other places ?

Reply:
Because they already have the `--runInBand` option which is equivalent to running with 1 worker, and both options are not compatible with each other, see: https://github.com/RequestNetwork/requestNetwork/blob/2c967592c9e295cf1f8ae6d93ecc0ecf8d66dacd/packages/request-node/package.json#L34